QuasarHelios2015/src/org/team1540/quasarhelios/AutoLoader.java

```diff
@@ -18,14 +17,15 @@ public static BooleanStatus create() {
 		a.setShouldBeRunning(b);
 		a.updateWhen(Igneous.globalPeriodic);
 		
-		Elevator.elevatorControl.setFalseWhen(BooleanMixing.onRelease(b));
+		Elevator.raising.setFalseWhen(BooleanMixing.onRelease(b));
+		Elevator.lowering.setTrueWhen(BooleanMixing.onRelease(b));
 		
 		return b;
 	}
 	
 	@Override
 	public void autonomousMain() throws AutonomousModeOverException, InterruptedException {
-		Elevator.elevatorControl.set(true);
+		Elevator.raising.set(true);
```

Comment:
Lowering should be set to false here, in case it was already lowering when the autoloader was activated